src/Microsoft.Azure.ServiceBus/Extensions/MessageInterOpExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.ServiceBus.Extensions
```

Comment:
In that case, perhaps namespace should be something like `Microsoft.Azure.ServiceBus.Interop`?  Not in the main space, but indicates what it stands for.